notadroid1007.md

```diff
@@ -48,7 +48,8 @@ These are the things I need to make my amazing project...
 | Battery connectors   | Owned                                                                                    | £ ---  |
 | Bluetooth Keyboard   | Owned                                                                                    | £ ---  |
 | Shipping             | ------------------------------- Shipping ----------------------------------------------- | £37.20 |
-| Total                |                                                                                          | £205.80|
+| Tax                  |                                                                                          | £4.10  |
+| Total                |                                                                                          | £209.90|
 
 
 ## £205.80 GBP is $246.57 USD
```

Comment:
```suggestion
## £209.90 GBP is $256.73 USD (approx.)
```

heyyyy thanks for including tax! after conversion you seem to be slightly above the 250USD grant limit - would you be able to cover the rest? thanks :)